src/components/AvatarWithIndicator.js

```diff
@@ -24,15 +25,12 @@ const propTypes = {
     /** To show a tooltip on hover */
     tooltipText: PropTypes.string,
 
+    /** All the user's policies */
```

Comment:
```suggestion
    /** The list of this user's policies (coming from Onyx) */
```

I feel like the previous comment was better

Reply:
Hmm, it's an object not a list, but I do like the additional detail. How about this?
`/** All the user's policies (from Onyx via withFullPolicy) */`